packages/apps-config/src/ui/logos/index.ts

```diff
@@ -68,6 +69,8 @@ import nodeTrustBase from './nodes/trustbase.png';
 import nodeUniarts from './nodes/uniarts.png';
 import nodeZenlink from './nodes/zenlink.svg';
 import nodeZero from './nodes/zero.svg';
+
```

Comment:
```suggestion
```

Reply:
i see!!! Blame me for not being careful ! Thanks!